server/mvcc/buckets/bucket.go

```diff
@@ -69,6 +69,7 @@ func (b bucket) IsSafeRangeBucket() bool { return b.safeRangeBucket }
 var (
 	MetaConsistentIndexKeyName = []byte("consistent_index")
 	MetaTermKeyName            = []byte("term")
+	MetaLocalVersionName       = []byte("local-version")
```

Comment:
Please document in: 

https://docs.google.com/document/d/1O2o1IApHWmSioXG3fez4eVlUHOrXICYGNVIzaqNS0IQ/edit?resourcekey=0-e6Iywgdkol0uiVBAaV1oww#heading=h.bapwseangl8w

I would rather call it 'storageVersion` or `backendVersion` or `fileFormatVersion`.